[ADD] fleet: new configurations doc

larm-odoo opened the pull request:
Adding new fleet documentation- this one is all the configurations in the config column.

robodoo commented:
[Pull request status dashboard](https://mergebot.odoo.com/odoo/documentation/pull/5763).

jcs-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,208 @@
+===================
+Fleet configuration
+===================
```
What about "configuration" only? it is in the "Fleet" category. But see also my suggestion in the main comment.

larm-odoo replied:
Hi @jcs-odoo - I have two documents currently written in github for fleet, and I am working on a third. So ultimately, they will all be nested under 'fleet', so I didn't want to make the default fleet doc be this one. 

I had done the format you suggested for other apps, where main doc I wrote covered 90% of things, but this time I was trying to chop the docs up into smaller digestible pieces, so that's why I tried to make separate docs for different items (addling a  vehicle, configurations, reports, repairs, etc).  If you don't think that's appropriate or a good idea, I can combine all the docs into one, but I was hoping to try and make the docs less overwhelming by having them separated out. Does that sound good?

I also know it means that once one doc is approved and posted, I will need to change the TOC so it looks correct with the newly added docs. I didn't want to wait to have the first doc be posted for review, have it go live, then add the rest, one at a time. That is why there are 2 (with another one on the way) posted now for review.  Sound good?

jcs-odoo replied:
Hi @larm-odoo 
You're right, it's better to have several pull requests rather than a massive one, at least in this documentation repository. Once the first PR is merged, you'll probably have a conflict on the other ones until you fix the toc tree. So that's fine :)

I agree this content only wouldn't make that much sense for the main page. Now that I see it again, I think my suggestion isn't better. "Configuration" alone is too vague (a bit like "misc." or "advanced"). Maybe "vehicle models and manufacturers"? That would be more representative. It's up to you :) 

Have a good day

jcs-odoo reviewed (commented):
Hi @larm-odoo 

If it fits the structure you have planned for this app's documentation, could you maybe move the configuration info to the app-level page (hr/fleet.rst)? Just a suggestion :)

Cheers :)

jero-odoo reviewed (changes requested):
@larm-odoo Nice work, just a few things to edit.

larm-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,208 @@
+Manufacturers
+=============
+
+Odoo Fleet comes pre-configured with sixty-six commonly used car manufacturers in the database,
```
If you look in Runbot, one of the manufacturers is for a bicycle, not a car. Then, if you make a DB using 15.0 form scratch, it is just the 66 manufacturers. So the test data in runbot is adding a bicycle which is not the default. This took me a while to figure out in the beginning since I had added in that bike manufacturer at first, but it's misleading in Runbot.

larm-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,208 @@
+******
+
+- :guilabel:`Can be requested`: Check this box if employees can request this model vehicle.
+- :guilabel:`Catalog Value (VAT Incl.)`: Enter the MSRP (Manufacturer's Suggested Retail Price) for
```
Great- added!

larm-odoo commented:
Just FYI- this is done in 15.2 runbot. When requesting a new vehicle, in this doc, the word is CREATE but in 15,0 it is NEW- just an FYI so you know. That's really the only difference in the versions for this doc, so I wanted to point that out.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+End Date Contract Alert
+-----------------------
+
+To help ensure no contracts lapse or vehicles are left without contracted drivers, Odoo has an
+alert in place to let the person responsible for the vehicle contract know when a contract is abut
+to expire. Enter the number of days before a contract is going to expire that an alert email will be
+sent.
```
```suggestion
To help ensure no contracts lapse or so no vehicles are left without contracted drivers, Odoo has an
alert in place to let the person responsible for the vehicle contract, know when a contract is about
to expire. Enter the number of days before a contract is going to expire, so that an alert email will be
sent.
```

This paragraph needs to be reworded. Last sentence doesn't make sense and first line is a bit confusing

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+to expire. Enter the number of days before a contract is going to expire that an alert email will be
+sent.
+
+.. note::
```
Do you cover this somewhere else?

larm-odoo replied:
Not in this doc- but I believe I also explain who the "Responsible" person is in the other doc as well.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+New Vehicle Request
+-------------------
+
+Set a limit on how many vehicle are in the fleet by setting a limit on requests. When using the
```
This is confusing.... the page says: 
`New Vehicle Request
An employee cannot request a new car if there are more than 
1,000
 existing cars availables.`
```suggestion
Set a limit on how many vehicles are in the fleet by setting a limit on requests. When using the
```

Doesn't say limit the fleet size-- but only deter folks from requesting new cars

larm-odoo replied:
True- I guess I saw this as a way to limit the size of the fleet, since people would not be able to request new cars if X old cars are already available - but I can see how that is a leap.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+-------------------
+
+Set a limit on how many vehicle are in the fleet by setting a limit on requests. When using the
+salary configurator, employees will not be able to request a new car if there is a specific amount
```
where does salary configuration come into play here? Where is this?

larm-odoo replied:
The option to request a new vehicle is only seen on the salary configurator. This is the only place an employee can request a new vehicle. I checked with Quentin in Belgium on a lot of these items since I wasn't sure about a lot of  these.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+-------------------
+
+Set a limit on how many vehicle are in the fleet by setting a limit on requests. When using the
+salary configurator, employees will not be able to request a new car if there is a specific amount
```
change amount to number

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+Configuration --> Manufacturers`.
+
+The list of manufacturers appear in alphabetical order. Each manufacturer's card lists how many
+specific models are configured for each particular manufacturer. Odoo comes with forty five
+pre-configured models for four major auto manufacturers.
```
example? you talk numbers twice here and its a little confusing.

larm-odoo replied:
I changed "for" to "from" and the manufacturers, so I think it's clearer now. 

Odoo comes with forty five pre-configured models from four major auto manufacturers: Audi, BMW, Mercedes, and Opel.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+
+To add a new manufacturer to the database, click :guilabel:`Create`. A manufacturer form will
+load. Only two pieces of information are needed, the name of the manufacturer, and the logo. Type
+the name of the manufacturer in the name field, and select an image to upload for the logo. When the
```
name is missing gui label

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+Vehicle Models
+==============
+
+Each manufacturer has a variety of models that they produce. When adding a vehicle to the fleet, it
```
```suggestion
Many manufacturers have a variety of models that they produce. When adding a vehicle to the fleet, it
```

larm-odoo replied:
It never occurred to me that there could be a manufacturer with ONE single model =D but I guess it could happen!

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+==============
+
+Each manufacturer has a variety of models that they produce. When adding a vehicle to the fleet, it
+is important to specify the vehicle being added. Odoo comes with pre-configured models for four
```
```suggestion
is important to specify the vehicle model being added. Odoo comes with pre-configured models for four
```

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+
+Each manufacturer has a variety of models that they produce. When adding a vehicle to the fleet, it
+is important to specify the vehicle being added. Odoo comes with pre-configured models for four
+manufacturers, so it is likely new models will need to be added to the database to correctly
+configure a fleet of vehicles.
```
This sentence doesn't make sense-- if they come preconfigured why do I have to add them?

larm-odoo replied:
I updated this so it's more clear- because there's only models configured for 4 manufacturers-most people (outside of Belgium) will need to add models.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create
+  & Edit`.
+- :guilabel:`Vehicle Type`: Select one of two preconfigured vehicle types, either :guilabel:`Car` or
+  :guilabel:`Bike`, from the drop-down menu. The vehicle types are hardcoded in Odoo and are
+  integrated with the *Payroll* application. Adding additional vehicle types is not possible as it
+  will affect payroll.
```
This makes me want to know why this is? Why is it integrated into payroll?

larm-odoo replied:
I added some more text here- vehicles are part of compensation/benefits/etc, so that's why there are parts of fleet that are firmly connected to Payroll.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+Information tab
+~~~~~~~~~~~~~~~
+
+Model
+*****
+
+- :guilabel:`Seats Number`: Enter how many passengers the vehicle can accommodate.
+- :guilabel:`Doors Number`: Enter the number of doors the vehicle has.
+- :guilabel:`Color`: Enter the color of the vehicle.
+- :guilabel:`Model Year`: Enter the year the vehicle was manufactured.
+- :guilabel:`Trailer Hitch`: Check this box if the vehicle has a trailer hitch.
```
This is covered in the other fleet doc? is this the same info?

larm-odoo replied:
This is in the new vehicle doc, yes. I noted on there that I'll let @StraubCreative decide if I should remove that section form that doc, or remove it form here and link to that doc, or keep them in both,

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+- :guilabel:`Can be requested`: Check this box if employees can request this model vehicle.
+- :guilabel:`Catalog Value (VAT Incl.)`: Enter the :abbr:`MSRP (Manufacturer's Suggested Retail
+  Price)` for the vehicle at the time of purchase or lease.
+- :guilabel:`C02 fee`: This value is specific to Belgian based companies only. If logged into a
+  Belgian company, this value is automatically calculated based on Belgian laws and regulations, and
+  cannot be modified. The value is based on the value entered in the :guilabel:`CO2 Emissions` field
```
any other European countries?

tiku-odoo replied:
let's test this out or ask an expert-- talking about Belgian based companies so much in your docs makes the software seem like it is just suited for that market, we are worldwide-- and can accommodate lots of different situations. Also this is best suited for a note block or something like that--- important

larm-odoo replied:
I did- I asked Quentin abut all of these fields because there was NO way I was able to figure that out myself! =D 

There are unfortunately a bunch of fields that are Belgian-specific or only appear in a Belgian localization. I notice that is the case for a bunch of areas in our HR docs.

larm-odoo replied:
I checked again- and the section doesn't even appear for non-Belgian companies So I updated this with an intro sentence.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+- :guilabel:`C02 fee`: This value is specific to Belgian based companies only. If logged into a
+  Belgian company, this value is automatically calculated based on Belgian laws and regulations, and
+  cannot be modified. The value is based on the value entered in the :guilabel:`CO2 Emissions` field
+  in the :guilabel:`Engine` section of the vehicle form. Modifying the :guilabel:`CO2 Emissions`
+  field will adjust the value in the :guilabel:`CO2 fee` field.
```
this should be put in an admonition block

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+  in the :guilabel:`Engine` section of the vehicle form. Modifying the :guilabel:`CO2 Emissions`
+  field will adjust the value in the :guilabel:`CO2 fee` field.
+- :guilabel:`Cost (Depreciated)`: Enter the monthly cost for the vehicle, which appears in the
+  salary configurator. This value impacts the gross and net salary of the employee who is assigned
```
again salary configuration is familiar to you, but I have no idea where this comes from

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+  field will adjust the value in the :guilabel:`CO2 fee` field.
+- :guilabel:`Cost (Depreciated)`: Enter the monthly cost for the vehicle, which appears in the
+  salary configurator. This value impacts the gross and net salary of the employee who is assigned
+  to the vehicle. This figure is depreciated over time.
```
where is it depreciated?

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+- :guilabel:`Cost (Depreciated)`: Enter the monthly cost for the vehicle, which appears in the
+  salary configurator. This value impacts the gross and net salary of the employee who is assigned
+  to the vehicle. This figure is depreciated over time.
+- :guilabel:`Total Cost (Depreciated))`: This value is the :guilabel:`Cost (Depreciated)` and the
+  :guilabel:`C02 fee` fields combined, and also is depreciated over time.
+
+.. note::
+   The :guilabel:`Cost (Depreciated)` does not depreciate automatically on the vehicle model, it
+   only depreciates based on the contract linked to a specific vehicle and not on the general model.
```
these are monthly figures and should be stated as such in bold in the doc

larm-odoo replied:
Added this to the top admonition block.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+Engine
+******
+
+- :guilabel:`Fuel Type`: Select the type of fuel the vehicle uses form the drop-down menu. The
+  options are :guilabel:`Diesel`, :guilabel:`Gasoline`, :guilabel:`Hybrid Diesel`, :guilabel:`Hybrid
+  Gasoline`, :guilabel:`Plug-in Hybrid Diesel`, :guilabel:`Plug-in Hybrid Gasoline`,
+  :guilabel:`CNG`, :guilabel:`LPG`, :guilabel:`Hydrogen`, or :guilabel:`Electric`.
+- :guilabel:`CO2 Emissions`: Enter the carbon dioxide emissions the vehicle produces in grams per
+  kilometer (g/km).
+- :guilabel:`CO2 Standard`: Enter the standard amount of carbon dioxide for a similar sized vehicle.
+- :guilabel:`Transmission`: Select the transmission type from the drop-down menu, either
+  :guilabel:`Manual` or :guilabel:`Automatic`.
+- :guilabel:`Power`: If the vehicle is electric or hybrid, enter the power the vehicle uses in
+  kilowatts (kW).
+- :guilabel:`Horsepower`: Enter the vehicle's horsepower in this field.
+- :guilabel:`Horsepower Taxation`: Enter the amount that is taxed based on the size of the vehicles
+  engine. This is determined by local taxes and regulations, and varies depending on the location.
+  It is recommended to check with the accounting department to ensure this value is correct.
+- :guilabel:`Tax Deduction`: This field auto-populates according to the engine specifications, and
+  cannot be modified. The percentage is based on the localization settings and local tax laws.
```
This is a repeat of the other doc. need to differentiate the two I am confused--- which populates which, one can be removed?

larm-odoo replied:
These are actually different forms- one is when adding a new vehicle, this is when adding a new model in general. It is really confusing because 80% of the form is identical- BUT the form name is actually different, so it is not the same. It just looks a lot like the other!

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+  options are :guilabel:`Diesel`, :guilabel:`Gasoline`, :guilabel:`Hybrid Diesel`, :guilabel:`Hybrid
+  Gasoline`, :guilabel:`Plug-in Hybrid Diesel`, :guilabel:`Plug-in Hybrid Gasoline`,
+  :guilabel:`CNG`, :guilabel:`LPG`, :guilabel:`Hydrogen`, or :guilabel:`Electric`.
+- :guilabel:`CO2 Emissions`: Enter the carbon dioxide emissions the vehicle produces in grams per
```
again over life per year? average?

larm-odoo replied:
it's i grams per kilometer- so it's an average but it's per kilometer (distace) not yearly or monthly (time based). I added it to the second entry to make it more clear.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+load. Enter the information in the vendor tab, then click :guilabel:`Save & Close` to add the vendor
+and close the window, or :guilabel:`Save & New` to add the current vendor and create another new
+vendor.
+
```
where do I go to see a list of all my vendors in fleet?

larm-odoo replied:
There isn't a separate list of vehicle vendors- they are the same vendors that you would use in the purchase app. So when you click ADD you see all the vendors in the database, from Azure Interior to BMW of Buffalo. I added a sentence to clarify this- since there is no dedicated vehicle vendor list.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+
+To view all the models currently set up, navigate to :menuselection:`Fleet application -->
+Configuration --> Model Category`. All models are displayed in a list view.
+
```
I would like to see examples

larm-odoo replied:
the last image is the model list- is this what you mean? The examples are bicycle, passenger car, minivan, passenger van, etc. It's just below.

tiku-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+
+To reorganize how the categories appear in the list, click on the up and down arrows to the left of
+the category to be moved, and drag the line to the desired position.
+
```
why should I order them?

tiku-odoo reviewed (commented):
@larm-odoo 

Good start...

I think this doc needs some more work though. I saw spelling errors after your team looked it over. I also have some suggestions. You repeat a section of this doc in the other doc (https://github.com/odoo/documentation/pull/5759) -- we should never be inputting info twice-- is it housed differently? can we prioritize and create an anchor?

The style seems cookie cut-- I would like to see better flow in these docs. Also keep in mind to write smart for 16/17. (save for example--- "if applicable, click save")

I would like to take a look at this doc again when you draft more changes.

Let me know if you have any questions. 

Thanks,
Tim :+1:

larm-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,234 @@
+=============
+Configuration
```
OK, Sam recommends Configuration menu, so I am going with that =)

larm-odoo commented:
Hi @tiku-odoo - I went through and addressed all your comments. I think everything should be clarified, but if not, let me know!
Ready for a second look.

tiku-odoo reviewed (commented):
@larm-odoo 

Looks much better! A couple of comments. It looks good. 

See comment: https://github.com/odoo/documentation/pull/5759#pullrequestreview-1674931749

I will approve once you can clarify the duplicate info. I know it's present on both forms; just confused why it's input twice.

larm-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,249 @@
+Set a limit on how many new vehicles are requested by setting a limit on requests based on fleet
+availability. When an employee is filling out the salary configurator form (after being offered a
+position), employees will *not* be able to request a new car if there is a specific number of cars
+available (vehicles that do not have a current driver assigned to them). Enter the number limit for
```
More than- I added an example block to make this more clear- as well as updated the text here.

larm-odoo commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,249 @@
+To reorganize how the categories appear in the list, click on the up and down arrows to the left of
+the category to be moved, and drag the line to the desired position. The order of the list does not
+affect the database in any way, however, it could be desired to view the vehicle categories in a
+specific order, for example, by size, or the numbers of passengers the vehicle can carry.
```
Yes- this isn't a default order, etc. This is just listing the possible models. I just like having things ordered, so if/where you can rearrange things, I tell people they can. I am the crazy person who would order these by seating capacity =D

larm-odoo commented:
Hi @tiku-odoo - I addressed everything and added an anchor in this doc (which will be linked to the other doc, the new vehicle doc). This doc needs to be published first so that the following doc can reference this one. Right now the other doc has a code error because it is linking to an anchor in this doc, which doesn't exist yet online. Please check/approve this one first, so I can have this published, then I will get the other doc published. Any issues with this let me know- thank you!

larm-odoo commented:
When this PR closes, get [[ADD] fleet: add a new vehicle](https://github.com/odoo/documentation/pull/5759#top)
#5759 merged next!

samueljlieber commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,254 @@
+   If the :guilabel:`New Vehicle Request` limit is set to 20 vehicles, and there are 25 vehicles
+   available, an employee would not be able to request a new vehicle. If there are only 10 cars
+   available, then the employee would be able to request a new vehicle.
+
```
```suggestion
.. _fleet/configuration/manufacturers:

```

samueljlieber commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,254 @@
+load. Only two pieces of information are needed, the :guilabel:`Name` of the manufacturer, and the
+logo. Type the name of the manufacturer in the name field, and select an image to upload for the
+logo. When the information is entered, click :guilabel:`Save`.
+
```
```suggestion
.. _fleet/configuration/models:

```

samueljlieber commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,254 @@
+==============
+
+Many manufacturers have a variety of models that they produce. When adding a vehicle to the fleet,
+it is important to specify the vehicle model being added. Odoo comes with pre-configured models from
+four manufacturers: Audi, BMW, Mercedes, and Opel. If a vehicle *other* than the pre-configured
+models from these manufacturers are part of a fleet, the model (and/or manufacturer) will need to be
+added to the database.
```
Added Eddy Merckx
```suggestion
it is important to specify the vehicle model being added. Odoo comes with pre-configured models five
major auto manufacturers: Audi, BMW, Eddy Merckx, Mercedes, and Opel. If a vehicle *other* than the
pre-configured models from these manufacturers are part of a fleet, the model (and/or manufacturer)
will need to be added to the database.
```

larm-odoo replied:
Eddy Merckx is actually a bicycle manufacturer, and the pre-loaded model is a bike, not a car. I am updating this to show that.

samueljlieber commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,254 @@
+
+The following models are pre-loaded in Odoo and do not need to be added to the database:
+
++-------+--------------+-----------+---------------+
+| AUDI  | BMW          | Mercedes  |     Opel      |
++=======+==============+===========+===============+
+| A1    | Serie 1      | Class A   | Agilia        |
++-------+--------------+-----------+---------------+
+| A3    | Serie 3      | Class B   | Ampera        |
++-------+--------------+-----------+---------------+
+| A4    | Serie 5      | Class C   | Antara        |
++-------+--------------+-----------+---------------+
+| A5    | Serie 6      | Class CL  | Astra         |
++-------+--------------+-----------+---------------+
+| A6    | Serie 7      | Class CLS | AstraGTC      |
++-------+--------------+-----------+---------------+
+| A7    | Serie Hybrid | Class E   | Combo Tour    |
++-------+--------------+-----------+---------------+
+| A8    | Serie M      | Class GL  | Corsa         |
++-------+--------------+-----------+---------------+
+| Q3    | Serie X      | Class GLK | Insignia      |
++-------+--------------+-----------+---------------+
+| Q5    | Serie Z4     | Class M   | Meriva        |
++-------+--------------+-----------+---------------+
+| TT    |              | Class R   | Mokka         |
++-------+--------------+-----------+---------------+
+|       |              | Class S   | Zafira        |
++-------+--------------+-----------+---------------+
+|       |              | Class SLK | Zafira Tourer |
++-------+--------------+-----------+---------------+
+|       |              | Class SLS |               |
++-------+--------------+-----------+---------------+
```
Added Eddy Merckx
```suggestion
+-------+--------------+-------------+-----------+---------------+
| AUDI  | BMW          | Eddy Merckx | Mercedes  | Opel          |
+=======+==============+=============+===========+===============+
| A1    | Serie 1      | SanRemo76   | Class A   | Agilia        |
+-------+--------------+-------------+-----------+---------------+
| A3    | Serie 3      |             | Class B   | Ampera        |
+-------+--------------+-------------+-----------+---------------+
| A4    | Serie 5      |             | Class C   | Antara        |
+-------+--------------+-------------+-----------+---------------+
| A5    | Serie 6      |             | Class CL  | Astra         |
+-------+--------------+-------------+-----------+---------------+
| A6    | Serie 7      |             | Class CLS | AstraGTC      |
+-------+--------------+-------------+-----------+---------------+
| A7    | Serie Hybrid |             | Class E   | Combo Tour    |
+-------+--------------+-------------+-----------+---------------+
| A8    | Serie M      |             | Class GL  | Corsa         |
+-------+--------------+-------------+-----------+---------------+
| Q3    | Serie X      |             | Class GLK | Insignia      |
+-------+--------------+-------------+-----------+---------------+
| Q5    | Serie Z4     |             | Class M   | Meriva        |
+-------+--------------+-------------+-----------+---------------+
| Q7    |              |             | Class R   | Mokka         |
+-------+--------------+-------------+-----------+---------------+
| TT    |              |             | Class S   | Zafira        |
+-------+--------------+-------------+-----------+---------------+
|       |              |             | Class SLK | Zafira Tourer |
+-------+--------------+-------------+-----------+---------------+
|       |              |             | Class SLS |               |
+-------+--------------+-------------+-----------+---------------+

```

samueljlieber commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,254 @@
+
+- :guilabel:`Model name`: Enter the model name in the field.
+- :guilabel:`Manufacturer`: Select the manufacturer from the drop-down menu. If the manufacturer is
+  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create
+  & Edit`.
```
Early line break
```suggestion
  not configured, type in the manufacturer and then click :guilabel:`Create` or :guilabel:`Create &
  Edit`.
```

samueljlieber commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,254 @@
+  integrated with the *Payroll* application since vehicles can be part of an employee's benefits.
+  Adding additional vehicle types is not possible as it will affect payroll.
+- :guilabel:`Category`: Select the category the vehicle is categorized under from the drop-down
+  menu. To create a new category, type in the category and then click :guilabel:`Create
+  (new category)`.
```
Early line break
```suggestion
  menu. To create a new category, type in the category and then click :guilabel:`Create (new
  category)`.
```

samueljlieber commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,254 @@
+  figure is depreciated over time according to local tax laws. The :guilabel:`Cost (Depreciated)`
+  does not depreciate automatically on the *vehicle model*, it only depreciates based on the
+  *contract* linked to a specific vehicle and not on the general model.
+- :guilabel:`Total Cost (Depreciated))`: This value is the :guilabel:`Cost (Depreciated)` and the
```
Double closing parenthesis 
```suggestion
- :guilabel:`Total Cost (Depreciated)`: This value is the :guilabel:`Cost (Depreciated)` and the
```

samueljlieber commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,254 @@
+.. image:: configuration/vendor.png
+   :align: center
+   :alt: Vendor form to fill out when adding a new vendor.
+
```
```suggestion
.. _fleet/configuration/categories:

```

samueljlieber reviewed (changes requested):
Hi @larm-odoo! Awesome job with the content in this PR, very thorough 👍

I do have a few suggestions after my technical review, which you can review below. 

Please tag me for another look once my suggestions have been addressed, thank you!

larm-odoo commented:
Thank you Sam for your help and review! This is ready for another look.

samueljlieber commented on content/applications/hr/fleet/configuration.rst:
```diff
@@ -0,0 +1,267 @@
+position), will *not* be able to request a new car if the number of existing cars is greater than
+the number specified in the :guilabel:`New Vehicle Request` field. Enter the specific number limit
+for existing available cars in this field.
+
```
Extra line 🙂
```suggestion
```

samueljlieber commented:
Hi @StraubCreative this PR is good to go 👍

StraubCreative reviewed (approved):
@robodoo r+
